src/tool_storage.rs

```diff
@@ -135,15 +136,35 @@ impl ToolStorage {
     }
 
     /// Install all tools from all reachable manifest files.
-    pub fn install_all(&self, trust: TrustMode) -> anyhow::Result<()> {
+    pub fn install_all(&self, trust: TrustMode, skip_untrusted: bool) -> anyhow::Result<()> {
```

Comment:
The `skip_untrusted` flag might be better handled by giving back the vec of errors to the caller of `install_all` here, but it didn't really look great when I tried and would leave some room for not properly handling those errors.